grails-app/services/org/broadinstitute/orsp/IssueService.groovy

```diff
@@ -446,4 +446,16 @@ class IssueService {
         props
     }
 
+    String getProjectType(String projectKey) {
+        String projectType
+        Issue issue = queryService.findByKey(projectKey)
+        if(issue != null) {
+            projectType = IssueType.valueOfName(issue.getType()).prefix.toLowerCase()
+            projectType
+        } else {
+            projectType = null
+        }
+        projectType
```

Comment:
This could be simplified with the null-safe operator. Something like this
```
String type = issue?.getType()
projectType = Optional.ofNullable(type).getOrElse("")
```
Which could be shortened to a one-liner if needed.